src/vm.rs

```diff
@@ -255,6 +263,7 @@ impl<T: GuestStagePagingMode> Vm<T> {
                 const_oid::db::rfc5912::ID_SHA_384,
             )
             .map_err(Error::AttestationManagerCreationFailed)?,
+            htimedelta: Wrapping(CSR.time.get_value()).neg().0,
```

Comment:
I think we actually want to latch at the point the first vCPU in the TVM is run. If we latch at TVM creation, then the host can effectively control the start time the TVM observes (which could possibly effect, e.g., RNG seeds).

Reply:
So you want to do it in tvm_finalize ? That means the delta would not include the time to create a TVM (initial configuration/memory conversion/) which may be significant. Once we have a confidential page pool in place, conversion wouldn't be factor though.

Reply:
Not `tvm_finalize` -- the first time the TVM is run, i.e. `tvm_vcpu_run(0)`. The intention is for the TVM to observe a start time of ~0.